doc/source/whatsnew/v1.5.0.rst

```diff
@@ -882,8 +881,7 @@ Bug fixes
 
 Categorical
 ^^^^^^^^^^^
-- Bug in :meth:`Categorical.view` not accepting integer dtypes (:issue:`25464`)
```

Comment:
Oops, sorry it appears the _original_ entries got deleted too. Could you restore them? (the only change in this file should be 1 new entry

Reply:
Yeah, I figured. That's why I was a bit confused. No worries!